src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
 import './MoviesList.scss';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard';
+import { MovieCardTypes } from '../types';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <>
+    {movies.map(movie => (
+      <MovieCard {...movie} />
+    ))}
+  </>
 );
+
+MoviesList.propTypes = {
```

Comment:
You could also create `MoviesListTypes` in `types.js`

Reply:
done